packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts

```diff
@@ -35,11 +35,13 @@ export const runEvaluationJob = async (job: Job<RunEvaluationJobData>) => {
       .find(evaluationId)
       .then((r) => r.unwrap())
 
-    await runEvaluation({
+    const { response } = await runEvaluation({
       documentLog,
       evaluation,
+      documentUuid,
     }).then((r) => r.unwrap())
 
+    await response
```

Comment:
put a comment so devs understand why we do this seemly useless await

Reply:
Better than that there is a test in case someone remove it. But yes, I'll add a comment

Reply:
Done